k8s/sonarqube/chart/sonarqube/templates/postgresql-deployment.yaml

```diff
@@ -28,6 +28,24 @@ spec:
         ports:
```

Comment:
Please split this into another PR, do not mix README and bug-fixing.

Reply:
Done